packages/web/components/place-limit-tool/index.tsx

```diff
@@ -0,0 +1,185 @@
+          baseAsset={swapState.baseAsset}
+          quoteAsset={swapState.quoteAsset}
+          baseBalance={swapState.baseTokenBalance}
+          quoteBalance={swapState.quoteTokenBalance}
+          onTokenSelect={(newDenom) => setBaseDenom(newDenom)}
+          disabled={false}
+          orderDirection={orderDirection}
+        />
+        <div className="px-4 py-[22px] transition-all md:rounded-xl md:py-2.5 md:px-3">
+          <div className="flex place-content-end items-center transition-opacity">
+            <div className="flex items-center gap-1.5">
+              <Tooltip
+                content={
+                  <div className="text-center">
+                    {t("swap.maxButtonErrorNoBalance")}
+                  </div>
+                }
+                disabled={!swapState.inAmountInput.notEnoughBalanceForMax}
+              >
+                <Button
+                  variant="outline"
+                  size="sm"
+                  className={classNames(
+                    "text-wosmongton-300",
+                    swapState.inAmountInput.isMaxValue &&
+                      !swapState.inAmountInput
+                        .isLoadingCurrentBalanceNetworkFee &&
+                      !swapState.inAmountInput.hasErrorWithCurrentBalanceQuote
+                      ? "bg-wosmongton-100/20"
+                      : "bg-transparent"
+                  )}
+                  disabled={
+                    !swapState.inAmountInput.balance ||
+                    swapState.inAmountInput.balance.toDec().isZero() ||
+                    swapState.inAmountInput.notEnoughBalanceForMax
+                  }
+                  isLoading={false}
+                  loadingText={t("swap.MAX")}
+                  classes={{
+                    spinner: "!h-3 !w-3",
+                    spinnerContainer: "!gap-1",
+                  }}
+                  onClick={() => swapState.inAmountInput.toggleMax()}
+                >
+                  {t("swap.MAX")}
+                </Button>
+              </Tooltip>
+            </div>
+          </div>
+          <div className="mt-3 flex place-content-between items-center">
+            <div className="flex w-full flex-col items-center">
+              <LimitInput
+                onChange={swapState.inAmountInput.setAmount}
+                baseAsset={swapState.inAmountInput.balance!}
+                tokenAmount={swapState.inAmountInput.inputAmount}
+                price={swapState.priceState.price}
+              />
+            </div>
+          </div>
+        </div>
+        <div className="mt-3 flex place-content-between items-center text-body1">
+          <div className="flex w-full flex-col">
+            <div>
+              <span
+                className={classNames(
+                  "w-full bg-transparent text-white-full focus:outline-none"
+                )}
+              >{`When ${swapState.baseDenom} price is at `}</span>
```

Comment:
Let's make sure to not forget localization.
There's some instructions in the web or root README for generating localizations with `yarn machine-translate` in the root of the repo.

Reply:
This looks like it's been changed in design so will hold off on this one, will update others